io.cpp

```diff
@@ -292,16 +334,24 @@ void collectFromNode(int toID, uint32_t nextCollectTime)
                 Serial.print(" rssi: "); Serial.println(rf95.lastRssi());
                 // TODO: send data to the API
                 printMessageData(from);
-                writeLogLine(from, id);
+                //writeLogLine(from, id);
```

Comment:
we should be clear in the documentation that logging is not currently being supported. I'd like to revisit this later -- I think we should support logging in general but locking down the API stuff I think is a higher priority